app/presenters/publishing_api/worldwide_corporate_information_page_presenter.rb

```diff
@@ -33,5 +33,9 @@ def links
         worldwide_organisation: [item.worldwide_organisation.content_id],
       }
     end
+
+    def document_type
```

Comment:
More of a musing than a real comment and I don't think it'll work because of the way that the classes are arranged but, it'd be nice if this could go at a higher level because I'd assume that there are a few presenters that need to have this exact method implementation

Reply:
Yeah, it could certainly be a default if we were inheriting from another class.

I'll leave as-is for now, but we may want to think about this in the future (e.g. as things like `BaseItemPresenter` are included in every presenter, so we may want that to be a default too.